reminders_sender/lambda_function.py

```diff
@@ -13,28 +13,51 @@
     REMINDER_ID_EXPONENT_MAX, REMINDER_NAME, REMINDER_DAYS, REMINDER_FREQUENCY,
     REMINDER_AMOUNT, DEFAULT_REMINDER_ID, DEFAULT_NAME, DEFAULT_AMOUNT,
     DEFAULT_FREQUENCY, DEFAULT_DAYS, DEFAULT_START_TIME_EACH_DAY, 
-    DEFAULT_END_TIME_EACH_DAY, DEFAULT_DELTA, DAYS_VOCAB, 
+    DEFAULT_END_TIME_EACH_DAY, DEFAULT_DELETED, DEFAULT_DELTA, DAYS_VOCAB, 
     UserReminder, ReminderMap)
 
 # https://github.com/eternnoir/pyTelegramBotAPI/issues/161#issuecomment-343873014
 bot = telebot.TeleBot(TELE_TOKEN, threaded=False)
 
+CONVERT_WEEKS = {
+    'Weekdays' : {0, 1, 2, 3, 4},
+    'Weekends' : {5, 6},
+    'Every day' : {0, 1, 2, 3, 4, 5, 6}
+}
+
+
+def if_alert(
+    day2remind: str, start: int, end: int, freq: int, rest: int, is_deleted: bool, 
+    hour_user: int, day_user: int) -> bool:
+        if (start <= hour_user <= end and (hour_user - start) % freq == 0 and 
+            rest > 0 and day_user in CONVERT_WEEKS[day2remind] and not is_deleted):
+            return True
+        return False
+
 
 def send_reminders() -> None:
     users = UserReminder.scan()
     for user in users:
-        hour_in_user = (datetime.now() + timedelta(hours=user.time_delta)).hour
+        hour_user = (datetime.now() + timedelta(hours=user.time_delta)).hour
+        day_user = (datetime.now() + timedelta(hours=user.time_delta)).weekday()
         user_reminders = []
-        print(f'hour_in_user={hour_in_user}')
+        print(f'hour_user={hour_user}')
```

Comment:
It would be good to learn about how to setup logging rather than using print statements.  That way, you won't need to comment out the log statements.

This is more of an advanced thing to do so print statements are fine for now.